### PR TITLE
2.6.1 release changelogs n stats

### PR DESCRIPTION
**Pull request by oleorhagen**
* 2.6.1 release logs and stats
* mender-binary-delta 1.1.2 logs

### CODE REVIEW

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 1]
```diff
@@ -1406,6 +1406,156 @@ New changes in workflows-enterprise since 1.3.0:
...
+New changes in mender-binary-delta since 1.1.0:
+
+* Probe U-Boot env before use to support libubootenv fw tools
+  ([MEN-4246](https://tracker.mender.io/browse/MEN-4246))
+* Remove harmless warning message about unhandled states.
+* Fix failed rollback status when bootloader is the one to roll back.
+* Detect mismatches between `mender_boot_part` and `RootfsPart(A|B)` variables.
+
+
+## Mender 2.6.1
```
Comment (lluiscampos) [inline 202.Release-information/01.Release-notes-changelog/docs.md:1424]: Shouldn't this be between 2.4.3 and 2.7.0? Alternatively just on top (as we usually do, prepend as we go), but inserting it here with no good criteria does not look right to me.

Comment (oleorhagen) [reply 202.Release-information/01.Release-notes-changelog/docs.md:1424]: Hmm, sure, but should we then place `2.7.0` at the top maybe?
Then `2.6.1`, and lastly `2.4.3`, to make it pretty

Comment (oleorhagen) [reply 202.Release-information/01.Release-notes-changelog/docs.md:1424]: @lluiscampos

Comment (lluiscampos) [reply 202.Release-information/01.Release-notes-changelog/docs.md:1424]: @oleorhagen I wouldn't change the existing ones, so I suggest either put this one between 2.7.0 and 2.4.3 or just in the first one (which is the regular process, we just prepend them as we update the docs).

**Review by lluiscampos — APPROVED** [review]
Either option is fine, this file won't be here for very much longer

**Pull request merged**